packages/SystemsThinkingWorkbench-Core.package/STWSimulation.class/instance/startSimulation.st

```diff
@@ -3,5 +3,7 @@ startSimulation
 
 	self setSimulationRunning: true.
 	self playButton label: 'Stop'.
+	self startStepping.
 
-	
\ No newline at end of file
+
```

Comment:
Space. The final frontier. These are the voyages of the starship Enterprise. Its ongoing ...